website/docs/artist_activity_feed.md

```diff
@@ -101,3 +101,18 @@ Attachment files are stored in the docker storage space `/storage/server/uploads
 
 There is currently no file size limit but we intend to introduce a default that can be configurable.
 :::
+
+### Reactions
+
+The platform now supports a new way of communicating: Reactions! Any user can now react to comments on the plaform by adding emoji reactions.
+Available reactions set is limited at the moment, but may be adjusted in the future.
+
+![Activity feed attachments](assets/activity_feed/activity-feed-reactions.png)
```

Comment:
Could we please have a screenshot that gives a better example of the feature being used in a realistic production environment. This helps answer the question of “why” and is good marketing material for inspiring whomever is reading it. It also gives us an opportunity to sneak in other nice features like the @mentions.

You can set the passwords on demo users in order to log into them to write different comments.

First comment - Artist

```
I have updated the textures and lighting to make the skin look more realistic. What do you think @{supervisor_name}?
```
1 thumbs up, 4 hearts and 2 tada all from other users.

Change status activity

```
Artist changed status from In Progress to Pending Review?
```

Second comment - Supervisor?
```
Starting to look really good now! Can we just make sure that we maintain the same brightness from @v002?
```
1 thumbs up from Artist

```
Supervisor changed status from Pending Review to Ready to Start?
```